src/hu_coordinates.c

```diff
@@ -71,12 +71,12 @@ static split_fixed_t SplitFixed(fixed_t x)
 
     result.negative = x < 0;
     result.base = x >> FRACBITS;
-    result.frac = x & (FRACUNIT - 1);
+    result.frac = x & FRACMASK;
 
     if (result.negative && result.frac)
     {
         result.base++;
-        result.frac = FRACUNIT - result.frac;
+        result.frac = FRACMASK - result.frac + 1;
```

Comment:
I think this was already correct before.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1763/commits/77526e0e64b3ecfcb1584d2a382c220ee1d9c67b